movo_robot/movo_bringup/launch/sensors/movo_sensors.launch

```diff
@@ -41,13 +41,21 @@
         <param name="timelimit" type="int" value="5" />
     </node>
         
+ 
+    <node pkg="ira_laser_tools" name="laserscan_multi_merger_camera" type="laserscan_multi_merger" output="screen"> 
```

Comment:
i think this node should run as part of nightingale_perception.
also i think the node should consume {base_scan_multi and camera/scan}.

If you do it this way, then the new merge node will always run regardless of if nightingale_perception is running and providing the camera/scan. this is wasteful and can be confusing.
If you put this node as part of nightingale_perception, then it will always have the camera/scan topic.

One way to think about is that movo1.launch should be able to run without nightingale-perception, but nightingale-perception will _always_ run with movo1.launch running

Reply:
yeah let me try that that does make sense thanks